proposal/color-4-new-spaces.md

```diff
@@ -2327,31 +2345,229 @@ plain CSS function named `"hsl"` that function is named `"hsla"` instead.
 ## Deprecated Functions
 
 Individual color-channel functions defined globally or in the color module are
-deprecated in favor of the new `color.channel()` function. That includes:
-
-* `color.red()`/`red()`
-* `color.green()`/`green()`
-* `color.blue()`/`blue()`
-* `color.hue()`/`hue()`
-* `color.saturation()`/`saturation()`
-* `color.lightness()`/`lightness()`
-* `color.whiteness()`
-* `color.blackness()`
+deprecated in favor of the new `color.channel()` function. Legacy global color
+functions are also deprecated. These functions always throw errors. During
+the deprecation process, they act as alias functions described below.
 
-Legacy global color functions are also deprecated:
+### `color.red()`, `red()`
```

Comment:
You don't really need to explicitly define this for both the module and non-module versions; you can just steal the terminology [from the existing spec](https://github.com/sass/sass/blob/main/spec/built-in-modules/color.md#complement): "This function is also available as a global function named `...`."